tests/bootstrap.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
+require_once( __DIR__ . '/../src/DocRaptor.php');
+require_once( __DIR__ . '/../src/Exception/MissingAPIKeyException.php');
```

Comment:
It's considered good practice to ommit it because it's optional and accidental whitespace after the closing tag would trigger an error when trying to modify headers.